### DISCUSSION TITLE
When are new releases typically cut?

### DISCUSSION THREAD

**Discussion by SalomonSmeke** [wezterm/wezterm#2627] [Q&A] [answered]
As title! I was wondering what the criteria are for cutting from nightly to the mainline releases. I am on nightly because of some of the more recent features, but wonder how to tell/when I should usually expect things to make their way over.

I noticed: https://github.com/wez/wezterm/discussions/969, https://github.com/wez/wezterm/discussions/1314, and https://github.com/wez/wezterm/milestones

They all seem to deal in "when is next release this time". Im more curious if there is some general guidelines about when they happen.

Obviously not a big deal if there isn't, I understand having a process for everything is onerous.

Cheers,
Salomon

**Comment by wez** [comment, marked as answer]
If I had no other conflicts for my time, I would like to cut a release roughly every 6 weeks, assuming there were no showstopper issues.

**Reply by SalomonSmeke** [reply]
Thank you!
